ast.go

```diff
@@ -1022,17 +1022,17 @@ func (node *CreateTable) String() string {
 	}
 
 	if node.StrictMode {
-		return fmt.Sprintf("CREATE TABLE %s (%s) STRICT", node.Table.String(), column)
+		return fmt.Sprintf("create table %s (%s) strict", node.Table.String(), column)
 	} else {
-		return fmt.Sprintf("CREATE TABLE %s (%s)", node.Table.String(), column)
+		return fmt.Sprintf("create table %s (%s)", node.Table.String(), column)
 	}
 }
 
 // StructureHash returns the hash of the structure of the statement.
 func (node *CreateTable) StructureHash() string {
 	cols := make([]string, len(node.Columns))
 	for i := range node.Columns {
-		cols[i] = fmt.Sprintf("%s:%s", node.Columns[i].Name.String(), node.Columns[i].Type)
+		cols[i] = fmt.Sprintf("%s:%s", node.Columns[i].Name.String(), strings.ToUpper(node.Columns[i].Type))
```

Comment:
To not break `StructureHash` compatibility I've added `strings.ToUpper`

Reply:
Ah ok, nice catch.